src/main/resources/com/hp/application/automation/tools/run/RunFromFileBuilder/config.jelly

```diff
@@ -38,6 +38,17 @@
     <f:entry title="Timeout" field="fsTimeout">
         <f:textbox name="runfromfs.fsTimeout" value="${instance.runFromFileModel.fsTimeout}"/>
     </f:entry>
+
+    <f:entry title="Run Mode" field="fsUftRunMode">
+        <select name="runPipeline.fsUftRunMode">
```

Comment:
This is page is for UFT only, users will be seeing this section only if they select UFT pipeline.
It would be meaningless to fold option field into another section.
I had confirmed this with Egor when this US was created, he agreed to put RunMode option here.